LabManual.md

```diff
@@ -205,3 +205,132 @@ Click **Search** to search for all documents.
+
+### Run the Bot ###
+
+Click F5 to run the sample code.  NuGet should take care of downloading the appropriate dependencies.  
+
+The code will launch in your default web browser in a URL similar to http://localhost:3979/.  
+
+> Fun Aside: why this port number?  It is set in your project properties.  In your Solution Explorer, double-click "Properties" and select the "Web" tab.  The Project Url is set in the "Servers" section.  
+
+![Bot Project URL](./assets/BotProjectUrl.jpg) 
+
+Make sure your project is still running (hit F5 again if you stopped to look at the project properties) and launch the Bot Framework Emulator.  Ensure that the Bot Url matches the port number that your code launched in above, and has api/messages appended to the end.  Now you should be able to converse with the bot.  
+
+TODO: picture of Bot Emulator
+
+### Add Intelligence to your Bot with LUIS ###
+
+Now let's expand this bot to integrate with our picture scenario.  We can give it some natural language capabilities with the [Language Understanding Intelligent Service](https://www.luis.ai/), or LUIS.  LUIS allows you to map natural language utterances to intents.  For our application, we might have several intents: finding pictures, sharing pictures, and ordering prints of pictures, for example.  We can give a few example utterances as ways to ask for each of these things, and LUIS will map additional new utterances to each intent based on what it has learned.  
+
+Navigate to https://www.luis.ai and sign in using your Microsoft account.  You should be redirected to a list of your LUIS applications at https://www.luis.ai/applications.  We will create a new LUIS app to support our bot.  
+
+> Fun Aside: Notice that there is also an "Import App" next to the "New App" button on [the current page](https://www.luis.ai/applications).  After creating your LUIS application, you have the ability to export the entire app as JSON, and check it into source control.  This is a recommended best practice so you can version your LUIS models as you version your code.  An exported LUIS app may be re-imported using that "Import App" button.  If you fall behind during the lab and want to cheat, you can click the "Import App" button and import the [LUIS model from the Finished section](./Finished/LUIS/CognitiveServicesTutorialLuisModel.json).  
+
+From https://www.luis.ai/applications, click the "New App" button.  Give it a name (I chose "PictureBotLuisModel") and set the Culture to "English".  You can optionally provide a description.  Click the dropdown to select an endpoint key to use, and if the LUIS key that you created on the Azure portal at the beginning of this lab is there, select it.  Then click "Create".  
```

Comment:
Might want something in here about ensuring that you're logged into luis.ai using the same account you used to create the keys earlier.